templates/apps/advancement/chain-configuration-dialog/chains.hbs

```diff
@@ -22,7 +22,7 @@
     </div>
     {{/if}}
     {{!-- TRAIT SELECTED CHOICES --}}
-    {{#if (eq node.advancement.type "trait")}}
+    {{#if (or (eq node.advancement.type "skill") (eq node.advancement.type "language"))}}
```

Comment:
Possible we might add more "trait" subtypes, so let's add an `isTrait` getter to the advancement. Alternatively, just add `isTrait: true` to the context.